CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [1.0-beta] - 2016-09-14
+### Added
+- App Engine Standard support ([#767](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/767))
```

Comment:
I think the way they want us calling this is: 'App Engine standard environment' support